### DISCUSSION TITLE
Idea: Multi-Language Support for Products and Attributes

### DISCUSSION THREAD

**Discussion by chrisoprea** [shopperlabs/framework#300] [Ideas]
Multiple language support for products and their attributes.

**Comment by mckenziearts** [comment]
Hey @chrisoprea, this is on our radar.

We're planning a dedicated **Translatable addon** (tracked in #461) that will handle translations for products, attributes, collections and other content models. The approach is inspired by how Medusa handles it: opt-in, kept out of core so stores that don't need it stay lean.

No official release date yet since it's still being scoped, but subscribe to #461 to get pinged when it starts moving.
